docs/source/developers/java/building.rst

```diff
@@ -46,7 +46,7 @@ repository:
 
 .. code-block::
 
-    $ git clone https://github.com/apache/arrow.git
+    $ git clone https://github.com/apache/arrow-java.git
     $ cd arrow
```

Comment:
```suggestion
    $ cd arrow-java
```

We need more similar changes.

BTW, we should move this documents to apache/arrow-java.

Reply:
Ah thank you for catching that.

> BTW, we should move this documents to apache/arrow-java.
Agreed, I cut https://github.com/apache/arrow-java/issues/472 in the java repo to migrate those docs. 

Once that issue is complete and the docs are hosted - will remove the docs here and change the java links in this repo's documentation to external links.

Reply:
Are we planning on hosting these separately?